index.html

```diff
@@ -14,6 +14,84 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
-
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    </head>
+    <body>
+    <header>
+    <!-- Nav bar -->
+    <img class="logo" src="img/karma-logo.svg" alt="coffee icon" />
+    <nav>
+    <ul class="nav__links" >
+    <li><b><a class="a-bold-1" href="#">Meet Karma</a></b></li>
+    <li><a href="#">How it works</a></li>
+    <li><a href="#">Store</a></li>
+    <li><a href="#">Blog</a></li>
+    <li><a href="#">Help</a></li>
+    <li><a href="#">Login</a></li>
+    </ul>
+    </nav>
+    </header>
+    
+    <!-- Main -->
+    <main>
+     <!-- Section -->
+    <section>
+    <div class="image__container">
+    <div class="inner__container">
+    <h1>Introducing Karma</h1>
+    <p>Bring WiFi with you, everywhere you go.</p>
+    <a class="btn" href="#">Learn More</a>
+    </div>
+    </div>
+    </section>
+    <section>
+    <div>
+    <h2>Everyone needs a little Karma</h2>
+    <div class="image__center">
+    <div>
```

Comment:
Buddy, I am trying to figure out what's this div all about here since it's empty, please review this area,

Reply:
Oh I see, I think it could be better if you had articles than keeping divs

Reply:
Hi Mathias, let me check it out bro. Thanks man

Reply:
Hi Kabelo,

Agree with Matthias a better suggestion to use articles.
From MDN [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) definition suggests selfcontained content and in this case multiple selfcontained content. Of course you can keep the divs but use in future. A snap of MDN articles in action,

![image](https://user-images.githubusercontent.com/65244840/174394583-31c06271-0478-4698-9aae-4cab8927cd4c.png)